.github/workflows/ci.yml

```diff
@@ -87,41 +87,43 @@ jobs:
       if: (matrix.platform.os == 'ubuntu-latest') && contains(matrix.platform.target, 'i686')
       run: sudo apt-get update && sudo apt-get install gcc-multilib
 
-    - name: Cache cargo-apk
-      if: contains(matrix.platform.target, 'android')
-      id: cargo-apk-cache
-      uses: actions/cache@v3
-      with:
-        path: ~/.cargo/bin/cargo-apk
-        # Change this key if we update the required cargo-apk version
-        key: cargo-apk-v0-9-7
-
-    - uses: dtolnay/rust-toolchain@master
-      if: contains(matrix.platform.target, 'android') && (steps.cargo-apk-cache.outputs.cache-hit != 'true')
+    - name: Install xbuild
+      uses: taiki-e/install-action@v2
+      if: contains(matrix.platform.target, 'android') || contains(matrix.platform.target, 'ios')
       with:
-        toolchain: stable
-
-    - name: Install cargo-apk
-      if: contains(matrix.platform.target, 'android') && (steps.cargo-apk-cache.outputs.cache-hit != 'true')
-      run: cargo install cargo-apk --version=^0.9.7 --locked
+        tool: xbuild@0.2.0
 
     - uses: dtolnay/rust-toolchain@master
       with:
         toolchain: ${{ matrix.toolchain }}${{ matrix.platform.host }}
         targets: ${{ matrix.platform.target }}
         components: clippy
 
+    - name: Install LLVM tools for Android
+      if: contains(matrix.platform.target, 'android')
+      run: sudo apt-get update && sudo apt-get install lld llvm
+
     - name: Check documentation
       run: cargo doc --no-deps $OPTIONS --document-private-items
 
-    - name: Build crate
-      run: cargo $CMD build $OPTIONS
+    - name: Build crate 
+      run: cargo build $OPTIONS
+
+    - name: Build and package crate for Android
+      if: contains(matrix.platform.target, 'android')
+      # TODO: Figure out why this build fails on Github Actions
+      run: x build -p ios-xbuild-target --platform android --arch arm64
```

Comment:
How does it work? Does it simply abort now or it builds, but not with `android` target?